java/ql/src/Metrics/Summaries/GeneratedVsManualCoverage.ql

```diff
@@ -0,0 +1,64 @@
+/**
+ * @id java/summary/generated-vs-manual-coverage
+ * @name Metrics of generated versus manual MaD coverage
+ * @description Expose metrics for the number of API endpoints covered by generated versus manual MaD models.
+ * @kind table
+ * @tags summary
+ */
+
+import java
+import semmle.code.java.dataflow.FlowSummary
+import utils.modelgenerator.internal.CaptureModels
+
+/**
+ * Returns the number of `DataFlowTargetApi`s with Summary MaD models
+ * for a given package and provenance.
+ */
+bindingset[package]
+private int getNumMadModeledApis(string package, string provenance) {
+  provenance in ["generated", "manual", "both"] and
+  result =
+    count(SummarizedCallable sc |
+      package = sc.asCallable().getCompilationUnit().getPackage().getName() and
+      sc.asCallable() instanceof DataFlowTargetApi and
+      (
+        sc.isAutoGenerated() and // "auto-only"
+        provenance = "generated"
+        or
+        (sc.hasProvenance(false) and not sc.hasProvenance(true)) and // "manual-only"
```

Comment:
Superfluous parentheses here and below.

Reply:
ah, sorry, I personally preferred the parentheses since they made it more readable to me. 
Removed in https://github.com/github/codeql/pull/11585/commits/3da2fb933bbc382c1ca9b62b5ad56c435bb00c5e.